### PR TITLE
Fixes Issue #5 & #10 & #21+ Live Stream Hot Fix

### PR DESCRIPTION
**Pull request by francistse23**
Fixes Kedyn/fusliez-notes#5 & Kedyn/fusliez-notes#10 & Kedyn/fusliez-notes#21

- Textarea in dark mode is now having a darker background
- Winning rate text is changed to white regardless of theme
- Win rate progress bar's red changed to a more muted tone #A62700
- Removed padding for player icon & removed opacity if Shows Players Name is false

### CODE REVIEW

**Comment by Kedyn** [comment]
Damn, awesome great work... I think we should discuss a bit about #13 that way we are on the same page before merging your modifications.

File: src/components/Controls.tsx [hunk 1]
```diff
@@ -83,7 +87,14 @@ export default function Controls(props: IControlsProps): JSX.Element {
         <Col>
           <h2>notes:</h2>
 
-          <Form.Control as="textarea" rows={20} />
+          <Form.Control
+            as="textarea"
+            rows={20}
+            style={{
+              backgroundColor: dark ? "dimgray" : "white",
+              color: dark ? "white" : " black",
```
Comment (chilblane) [inline src/components/Controls.tsx:95]: In the future, we should unify the styling of the textarea with the other "normal" text inputs, which we're currently using for the number of games played and won.

Comment (francistse23) [reply src/components/Controls.tsx:95]: yeah it wasn't the best solution. was trying to push the changes out asap.

Kedyn opened a new issue Kedyn/fusliez-notes#13 to discuss about the project structure and how to approach managing styling files and whatnot

I guess once that's settled we can look into unifying things so it isn't all over the place

**Comment by francistse23** [comment]
@Kedyn just fixed something based on the feedback from leslie and chat. please merge it if it's good. thanks!

**Comment by Kedyn** [comment]
All cool, but the player colors on blue and purple look a bit wrong... so we going with @chilblane hot fix

**Pull request closed without merging**